build/ci/phase-template.yml

```diff
@@ -40,9 +40,6 @@ phases:
     - ${{ if eq(parameters.name, 'MacOS') }}:
       - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
         displayName: Install runtime dependencies
-    - powershell: $(Build.SourcesDirectory)/build/PatchVSTestConsole.ps1
-      displayName: Hot patch vstest.console
```

Comment:
📝 I'm testing without this change, _hoping_ that the cause of coverage problems with 2.6.0 was not related to this specific change.

Reply:
Removed this change. We can bring it back if problems start occurring.